README.md

```diff
@@ -8,7 +8,45 @@
 
 This repo defines the web interface for the DUNE Process Manager.
 
-## For developers
+## Running the App
+
+To run with a demo version of the drunc process manager, run it with docker compose:
+
+```bash
+docker compose up
+```
+
+It can take a few moment for the services to boot but the application should then be
+available in the browser at <http://localhost:8000>.  Authentication is required to work
```

Comment:
The link checker is never happy with `localhost`. This needs to be exlcuded.

Reply:
Ugh, well that should be the default behaviour... I've added a config file to ignore all references to it.